engine/src/main/java/com/datatorrent/stram/engine/StreamingContainer.java

```diff
@@ -318,7 +319,8 @@ public static void main(String[] args) throws Throwable
       logger.error("Fatal {} in container!", (e instanceof Error) ? "Error" : "Exception", e);
       /* Report back any failures, for diagnostic purposes */
       try {
-        umbilical.reportError(childId, null, ExceptionUtils.getStackTrace(e));
+        LogFileInformation logFileInfo = LoggerUtil.getLogFileInformation();
```

Comment:
should LogFileInformation be fetched prior to logging error on line 319? Is it necessary to check that `logger` level is ERROR or above?